### ISSUE TITLE
XSD Validation Line Number

### ISSUE THREAD

**Issue by charlesgiroux** [lestrrat-go/libxml2#27]
Hi,

Is there a way to get the line numbers of the errors in the XML file when validating against an XSD?

Thanks,
Charles

**Comment by lestrrat** [comment]
It's not found in the errors returned by `Errors()` ?

**Comment by charlesgiroux** [comment]
Not from what I've seen. 

It only returns a description of the error. For example "error: Element '{http://mynamespace.com}MyElement': [facet 'minLength'] The value has a length of '2'; this under runs the allowed minimum length of '3'.

**Comment by Srae** [comment]
Hello, 
I'm encountering the exact same issue :(.
Do you think it could be added to the SchemaValidationError struct ?

**Comment by lestrrat** [comment]
Well, I have warnocked you guys pretty badly. I apologize.
A quick googling around hinted that other language binding users are having similar issues, but I couldn't find a particularly useful response. 
I also found that xmllint reports line numbers -- which probably means that there is an alternate way of getting the line numbers, and maybe we should be looking at the source code for xmllint for hints.

Having said that, as of this moment, I don't have much tuits for this, and if anybody can help me that would be great.
